js/themes-vendor/src/main/js/web-crypto-shim.js

```diff
@@ -1,4 +1,4 @@
-import { sha256 } from "@noble/hashes/sha256";
+import { sha256 } from "@noble/hashes/sha2";
```

Comment:
Rollup can probably resolve this, but [the changelog](https://github.com/paulmillr/noble-hashes/releases/tag/2.0.0) for `@noble/hashes` states the extension is mandatory:

> `.js` extension must be used for all modules 


```suggestion
import { sha256 } from "@noble/hashes/sha2.js";
```

This matches with what [exported from](https://github.com/paulmillr/noble-hashes/blob/d30e0707258f4cf0d4fb5dd6062436f8c1e997eb/package.json#L50) the package.